script.js

```diff
@@ -0,0 +1,178 @@
+// Assignment Code
+var generateBtn = document.querySelector("#generate");
+
+// Arrays
+const numArray = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
+const specialArray = [
+  "!",
+  "£",
+  "$",
+  "%",
+  "^",
+  "&",
+  "*",
+  "(",
+  ")",
+  "_",
+  "-",
+  "+",
+  "=",
+  "?",
+  "@",
+  "#",
+  "~",
+  "|",
+  "/",
+  ".",
+  ",",
+  "<",
+  ">",
+  '"',
+  ":",
+  " ",
+  ";",
+  "`",
+  "¬",
+];
+const lowerCaseArray = [
+  "a",
+  "b",
+  "c",
+  "d",
+  "e",
+  "f",
+  "g",
+  "h",
+  "i",
+  "j",
+  "k",
+  "l",
+  "m",
+  "n",
+  "o",
+  "p",
+  "q",
+  "r",
+  "s",
+  "t",
+  "u",
+  "v",
+  "w",
+  "x",
+  "y",
+  "z",
+];
+
+// convert lower case to uppercase array
+const upperCase = lowerCaseArray.join("").toUpperCase();
+const upperCaseArray = upperCase.split("");
+
```

Comment:
It took me a minute to find out where the uppercase array was. I've not seen it done this way before and I probably wouldn't of thought of doing it this way myself. And it works, so that's good. The only thing I'd say is be wary of doing things like this, as if there was an error in the lower case array, that would also go to the upper case array by default. Apart from that, this is fine.